packages/mermaid/src/diagrams/user-journey/journeyRenderer.ts

```diff
@@ -39,14 +41,18 @@ function drawActorLegend(diagram) {
       text: person,
       textMargin: conf.boxTextMargin | 5,
     };
-    svgDraw.drawText(diagram, labelData);
 
+    const textElement = svgDraw.drawText(diagram, labelData);
+    const textLength = textElement.node().getBBox().width;
+    if (textLength > maxWidth) {
+      maxWidth = textLength;
+    }
```

Comment:
Not blocker: We should keep this dynamic width part, but should we split the text into more lines, if it's longer that a longer max limit?